### PR TITLE
Transpiler - Cython Parser

### PR DESCRIPTION
**Pull request by cyrillemidingoyi**
transpiler/Parser  : Parse Cython source code into a parse tree.
tests_transpiler : an example of cython code parsed and the AST generated

### CODE REVIEW

File: src/pycropml/transpiler/Parser.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import os
```
Comment (pradal) [inline src/pycropml/transpiler/Parser.py]: Please, document the code. This is mandatory due to code complexity and maintenance.

What this module do?
What are the role of options (doc needed).
...

File: tests_transpiler/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (pradal) [inline tests_transpiler/__init__.py]: test/transpiler may be better

**Comment by cyrillemidingoyi** [comment]
Parser code commented.
I will comment the other codes and push them

**Comment by cyrillemidingoyi** [comment]
@pradal 
Hi, please access to the transpilation package structure on the wiki

**Pull request merged**